nexus/src/db/queries/network_interface.rs

```diff
@@ -1434,12 +1780,18 @@ mod tests {
             .instance_create_network_interface_raw(&opctx, interface.clone())
             .await
             .expect_err("Should not be able to insert more than 8 interfaces");
-        assert!(matches!(result, NetworkInterfaceError::NoSlotsAvailable,));
+        assert!(matches!(
+            result,
+            InsertNetworkInterfaceError::NoSlotsAvailable,
+        ));
 
         db.cleanup().await.unwrap();
         logctx.cleanup_successful();
     }
 
+    #[tokio::test]
+    async fn test_instance_network_interface_delete() {}
```

Comment:
Missing test body here?

Reply:
Good catch thanks. I added checks in the integration tests, since this now relies on records in the instance table as well. I'll remove this one.

Reply:
Note to self. The _insertion_ queries in this module will also need to become more like integration tests when resolving #1134. The instance table isn't currently referenced in the query itself, but will be to avoid that race.